templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,63 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+
+    # This function waits for ip address of br-ex to be bindable only in case of ipv6
+    # This is workaround for OCPBUGS-673 as it will not allow starting crio
+    # before address is bindable
+    try_to_bind_ipv6_address() {
+      # Retry for 1 minute
+      retries=60
+      until [[ ${retries} -eq 0 ]]; do
+        ip=$(ip -6 -j addr | jq -r "first(.[] | select(.ifname==\"br-ex\") | .addr_info[] | select(.scope==\"global\") | .local)")
+        if [[ "${ip}" == "" ]]; then
+          echo "No ipv6 ip to bind was found"
+          break
+        fi
+        random_port=$(shuf -i 49152-60999 -n 1)
```

Comment:
nit: I would actually have used something like 50000-60000 so as to not to be pegged at the boundaries of known ranges,  but I am fine with this in any case.